hoa2dot/grammars/hoa.lark

```diff
@@ -34,23 +34,27 @@ label:  "[" label_expr "]"
 
 
 state_conj : INT | state_conj "&" INT
-label_expr : boolean_label_expr | atom_label_expr | alias_label_expr | not_label_expr
-           | "(" label_expr ")"
-           | and_label_expr
+label_expr : "(" label_expr ")"
            | or_label_expr
-or_label_expr.2: label_expr "|" label_expr
-and_label_expr.1: label_expr "&" label_expr
+           | and_label_expr
+           | not_label_expr
```

Comment:
I don't think this precedence rule is 100% correct, but let's stick with it until either we get news from Lark or we found other problems